### PR TITLE
1 update description

### LINKED ISSUE #1: Update the challenge Description, Specify Delivery format

### PR DESCRIPTION
**Pull request by mmziyad**
fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+    * Latitude and Longitude in floating point format (second and third columns, respectively)
+    * Each row of the CSV can be considered as a single user tracking event.
+
+## Task
+
+Your application will need to do the following:
+  
+* Parse an input data-source containing events with the same fields as in the sample csv: user-id and geo-coordinates.
+
+* Geo-distance calculation: Take the users's coordinates and compare this with the set of airport coordinates to get the
```
Comment (fpopic) [inline README.md]: shouldn't it be users' ?

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
+You have a free hand in choosing the environment in which the code is developed and demonstrated.
+
+## Licensing 
+
+* `user-geo-sample.csv.gz`
+
+The longitude, latitude data in this sample was taken from a data-set provided
+by Maxmind inc.
+
+This work is licensed under the Creative CommonsAttribution-ShareAlike 4.0 International License.
```
Comment (fpopic) [inline README.md]: This work is licensed under the [Creative CommonsAttribution-ShareAlike 4.0 International License](http://creativecommons.org/licenses/by-sa/4.0/).

**Review by fpopic — APPROVED** [review]
Just typos.

File: README.md (hunk 2 above)
Comment (mmziyad) [reply README.md]: I have updated all links in Markdown style

**Review by ghost — APPROVED** [review]

**Pull request merged**